Dynavity/Dynavity/model/CanvasElementProtocol.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import CoreGraphics
+
+protocol CanvasElementProtocol {
+    var id: UUID { get set }
+    var position: CGPoint { get set }
+    var text: String { get set }
+    var visualID: String { get }
```

Comment:
What's the difference between visualID and id? (or rather, why is there a need for a visuaIID?)

Reply:
`visualID` here uses the UUID and text for this canvas element to create a unique ID, you're right it's not totally necessary since `UUID()` randomly generates a UUID with 4 random bytes. Was just a precaution! 

I think WL or Ian can change this as they deem fit for the actual canvas elements.

Reply:
I see, thanks!